.vscode/settings.json

```diff
@@ -2,5 +2,8 @@
     "editor.defaultFormatter": "ms-python.autopep8",
     "pylint.args": [
         "--disable=E0015"
-    ]
-}
\ No newline at end of file
+    ],
+    "[javascript]": {
+        "editor.defaultFormatter": "ms-vsliveshare.vsliveshare"
```

Comment:
I don't think this was intended to be part of the PR (since this does not seem like a general-purpose code formatter). Please double-check that this change is not staged for future commits. Maybe this should be part of the ignore instead since everyone already has this file, and I don't see any reason we would modify it for everyone (other than to add another formatter).

Reply:
Do I need to remove these lines of code and make another commit? or are you able to just ignore these changes?

Reply:
Could you add the .vscode directory to the .gitignore file and push? Removing the file or ignoring it would both require a push from someone. As far as I know, I'm not able to select specific parts of a PR to merge.